### ISSUE TITLE
TODO: Incorporate Accessibility best practices

### ISSUE THREAD

**Issue by mhairston** [vhl/html-style-guide#5]
They should be added to the markup guidelines here rather than put in their own document -- a11y has to be part of the process, not an add-on.

Good summary:
https://www.marcozehe.de/2015/12/14/the-web-accessibility-basics/
